app/src/main/res/layout/migrate_wallet_complete_fragment.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
could this not also be merged with create_wallet_complete_fragment.xml ?

Reply:
👍